src/screens/Addresses/Contact/EditContactScreen.tsx

```diff
@@ -101,13 +102,13 @@ const EditContactScreen = ({ navigation, route: { params }, ...props }: EditCont
   }
 
   return (
-    <Screen {...props}>
+    <ScrollScreen hasHeader {...props}>
```

Comment:
As mentioned in Slack as well: This doesn't need to be a scroll screen

Reply:
My main reason to do this is that on iOS all screens are scrollable. Check for instance the Wifi section in the settings (when wifi is off). Content is super short but the screen is scrollable.

Reply:
Following the same principle, all modals should also contain a scroll screen (not only the ones that render lists using `FlatList`. Correct?